mollie-payments-for-woocommerce/includes/mollie/wc/plugin.php

```diff
@@ -467,6 +502,11 @@ public static function getPluginUrl ($path = '')
     	return M4W_PLUGIN_URL . $path;
     }
 
+    public static function getPluginPath($path = '')
+    {
+        return M4W_PLUGIN_DIR . $path;
```

Comment:
If this method remains here at all, then the plugin dir is probably something that should be injected.

Reply:
@XedinUnknown That class could need a deeper look and probably a refactor, but not for now.
I just keeping the same code style that already exists.

Reply:
Looks like nothing gets injected anywhere, so we agree to keep it that way for now.